src/app/app.component.ts

```diff
@@ -70,13 +71,16 @@ export class AppComponent implements OnInit {
     this.runningCommands[commandId] = null;
   }
 
-  startCheckingCommand(commandId: string): void {
+  startCheckingCommand(commandId: string, isServeCommand): void {
```

Comment:
boolean

Reply:
you don't really need `isServeCommand` in this method.
just do
```typescript
if (isServeCommand) {
    this.serveCommandId = commandId;
}
```
in `sendCommand` method

Reply:
If we do that, we will need to use the same implementation of this function to clear `commandId` in case of an error while running `ng serve`